pipeline/scripts/5/0/tier-3/test-rados-scheduled-scrub.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Ceph RADOS tests: Scheduled scrub."
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.0"
+test_suite="suites/pacific/rados/tier_3_rados_scrub.yaml"
```

Comment:
Not related but it would be good if we could rename the suite according to our guideline. That is, `tier-3_rados_test-scrub.yaml`